package.json

```diff
@@ -20,7 +20,8 @@
     "start": "next",
     "test:e2e": "playwright test",
     "test:watch": "jest --watch",
-    "test": "jest"
+    "test": "jest",
+    "typecheck": "tsc -p ."
```

Comment:
Added this for local use currently. We could probably add a PR check for this though! I noticed some other projects do it.

Reply:
I dig this, nice and convenient and +1 on adding a PR check or pre-commit verification hook or something 💯